docs/content/configuration-language.mdx

```diff
@@ -261,20 +261,21 @@ The `folder` and `document` _type definitions_ each have five _relations_: `pare
 
 :::
 
-### The Direct Relationship Keyword
+### The Direct Relationship Type Restrictions
 
-`self`, when used in the context of the <ProductConcept section="what-is-a-relation-definition" linkName="relation definition" /> allows <ProductConcept section="what-are-direct-and-implied-relationships" linkName="direct relationships" /> to the objects of this type. The absence of `self` disallows these direct relationships.
+`[<type1>,<type2>]`, when used in the context of the <ProductConcept section="what-is-a-relation-definition" linkName="relation definition" /> allows <ProductConcept section="what-are-direct-and-implied-relationships" linkName="direct relationships" /> to the objects of these specified types. The absence of `[]` disallows these direct relationships.
 
 :::info
 
-`self` in the <ProductName format={ProductNameFormat.ShortForm}/> DSL translates to `this` in the <ProductName format={ProductNameFormat.ShortForm}/> API syntax.
+`[]` in the <ProductName format={ProductNameFormat.ShortForm}/> DSL translates to `this` in the <ProductName format={ProductNameFormat.ShortForm}/> API syntax.
 
 :::
 
 For example, let's take a closer look at the `team` type.
 
 <AuthzModelSnippetViewer
   configuration={{
+    schema_version: '1.1',
```

Comment:
This would result in incorrect JSON though.

Reply:
The JSON will still be correct (it is a snippet)

The result is 
```
{
  "type": "team",
  "relations": {
    "member": {
      "this": {}
    }
  },
  "metadata": {
    "relations": {
      "member": {
        "directly_related_user_types": [
          {
            "type": "user"
          },
          {
            "type": "user:*"
          },
          {
            "type": "team",
            "relation": "member"
          }
        ]
      }
    }
  }
}
```


<img width="2032" alt="Screen Shot 2023-01-06 at 5 41 52 PM" src="https://user-images.githubusercontent.com/10730463/211112152-c8ed7dbb-aa7d-4ba6-80ec-152e90000143.png">

Notice the component will also remove the schema version for JSON as well.